pilot/pkg/features/pilot.go

```diff
@@ -151,6 +151,12 @@ var (
 		"UseRemoteAddress sets useRemoteAddress to true for side car outbound listeners.",
 	).Get()
 
+	UseMaglevForConsistentHash = env.RegisterBoolVar(
```

Comment:
can be removed

Reply:
API changes are not merged yet to main repo. I will fix it after api is reflected here.

Reply:
why is it still here since the API is merged

Reply:
API changes have not been reflected to Istio/Istio. I am waiting for it to happen so that I can change this.  https://github.com/istio/istio/pull/40479/files is merged today. So I will rebase and fix it.

Reply:
What change do u mean? Not https://github.com/istio/api/pull/2434?

Reply:
That change only was not updated to main repo because it was failing lint validation for deprecated field in that PR. Automatic API did not happen. https://github.com/istio/istio/pull/40479/files manually updated the API.

Reply:
So let's wait for it. I believe eventually we meant to implement it rather than this env flag.